[IMP] Maintain: Add process to delete account and clean-up

tiku-odoo opened the pull request:
Added section per Documentation Task (https://www.odoo.com/web#id=3129179&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form) 

Now includes the process of deleting the Odoo.com account from the admin console. 

I also cleaned up the document to remove 2nd-person references and streamline material.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/3443).

tiku-odoo commented:
@jero-odoo This doc is ready for your review when you have a moment. Thank you.

jero-odoo commented on content/administration/maintain/odoo_online.rst:
```diff
@@ -1,22 +1,43 @@
+Odoo Online user management
+===========================
+
+To delete the Odoo account, access the delete option by clicking on the
```
```suggestion
To delete an Odoo account, access the delete option by clicking on the
```

jero-odoo reviewed (commented):
Just the note about adding additional information about what happens after you try to delete the account, and whether additional steps are required.

tiku-odoo commented:
CC: @elm-odoo -- When you have some time can you review this doc? I've added in Lines 4-19 and cleaned up the rest of the document. Can you review for accuracy? 

Thanks, 
Tim

tiku-odoo commented on content/administration/maintain/odoo_online.rst:
```diff
@@ -1,22 +1,43 @@
+.. danger::
+   Deleting an Odoo account is irreversible. Take caution when performing this action as the
+   Odoo.com account will not be retrievable.
+
```
I've added a screenshot in and some additional steps. I will send this to secondary peer review. Thanks for your help on this! 👍

jcs-odoo reviewed (changes requested):
Hi @tiku-odoo 

This doc is about Odoo Online database management (that's why it's in install/maintain)

I disagree with this PR as it changes the title of the doc and puts the database management info in second. I also think the topic "odoo.com account management" doesn't belong here. 

It doesn't fit misc/users perfectly either since the info is about odoo.com and not a database, but I guess it's already better? Unless you create a new page or category specifically for odoo.com accounts, including the security info and the customer portal from a user point of view?

tiku-odoo commented:
@brse-odoo Hey Brandon-- this doc is ready for your review when you have a chance 👍 Thanks for your help on this doc!

brse-odoo commented on content/administration/maintain/odoo_online.rst:
```diff
@@ -59,27 +64,27 @@ upgrade to the latest version (e.g., 13.0 to 15.1).
-========
+--------
 
 Download instantly a ZIP file with a backup of the database.
```
```suggestion
Instantly download a ZIP file with a backup of the database.
```

brse-odoo commented on content/administration/maintain/odoo_online.rst:
```diff
@@ -2,21 +2,26 @@
    :alt: Clicking on the gear icon opens the drop-down menu.
 
-Open the drop-down menu next to the database you want to manage by clicking on the gear icon.
+Open the drop-down menu next to the database that changes will be made on by clicking on the gear
```
Should the gear icon be a guilabel?

tiku-odoo replied:
I was told by Jonathan that it isn't a guilabel.

tiku-odoo commented:
@StraubCreative This PR on the deletion process is ready for your review. 

I've created a new doc under Users & Companies for the Odoo.com account deletion and cleaned up the Odoo Online Database Management. When you have a moment can you review both documents? 

Thanks in advance for your help reviewing 👍

samueljlieber commented:
Updates in ccb3fa0 
- fixed build errors
    - `:menuselection:` formatting
    - added an extra line at the end of `delete_account.rst`
    - fixed a few lines that broke early of 100 chars

StraubCreative commented on content/applications/general/users/delete_account.rst:
```diff
@@ -0,0 +1,35 @@
+===========================
+Odoo.com Account Management
```
Wondering—
If this doc is exclusively about deleting an Odoo account, then wouldn't we want to name it as such? Or are we planning to add more content to this doc later?

tiku-odoo replied:
I've renamed the doc: Deleting Odoo.com Account. Great suggestion.

StraubCreative commented on content/applications/general/users/delete_account.rst:
```diff
@@ -0,0 +1,35 @@
+Deletion Process
+----------------
+
+Click on the user icon and select :guilabel:`My Account`. Then click on :guilabel:`Edit Security
+Settings`.
+
+Next, under the :guilabel:`Delete Account` section, click on the :guilabel:`Delete Account` button.
```
Assuming this doc is *only* about account deletion (and we're not planning to revise the overview later to make room for new content), then either the Overview or the first sentence should be revised since it's redundant.

Suggestion below:

```suggestion
On the :guilabel:`Security Settings` page, scroll to the :guilabel:`Delete Account` section, and
then click on the :guilabel:`Delete Account` button.
```

tiku-odoo replied:
Noted, I've reduced the number of sentences. Thanks.

StraubCreative commented on content/applications/general/users/delete_account.rst:
```diff
@@ -0,0 +1,35 @@
+Next, under the :guilabel:`Delete Account` section, click on the :guilabel:`Delete Account` button.
+
+.. danger::
+   Deleting an Odoo account is irreversible. Take caution when performing this action as the Odoo.com
+   account will not be retrievable.
```
Break earlier on line 22
```suggestion
   Deleting an Odoo account is irreversible. Take caution when performing this action as the
   Odoo.com account will not be retrievable.
```

StraubCreative commented on content/applications/general/users/delete_account.rst:
```diff
@@ -0,0 +1,35 @@
+   Deleting an Odoo account is irreversible. Take caution when performing this action as the Odoo.com
+   account will not be retrievable.
+
+Upon clicking the :guilabel:`Delete Account` button Odoo will populate a pop-up window verifying the
+change to delete the account.
```
Sentence reads weird imo.

Wording suggestion to simplify it:
```suggestion
Upon clicking the :guilabel:`Delete Account` button, a pop-up window will appear, asking you to
confirm the account deletion.
```

tiku-odoo replied:
👍 All changed! Thanks.

StraubCreative reviewed (changes requested):
Hi @tiku-odoo 
Nice work. Just a couple quick suggestions.
See below.

tiku-odoo commented:
@StraubCreative This doc has been changed per your review. Thanks for your help on this doc. It's ready for a final pass.

tiku-odoo commented:
@jcs-odoo Thanks for the information, I've made the headline changes to both docs (delete_account.rst & odoo_online.rst. 

@StraubCreative This doc is ready for your review. 

Thanks, Tim 👍

samueljlieber commented:
Rebased to latest, fixed build errors, and fixed merge conflict in https://github.com/odoo/documentation/commit/40fc0528f47922d085d3666814d8b7177423b913

StraubCreative commented on content/administration/maintain/odoo_online.rst:
```diff
@@ -35,22 +37,22 @@ Several actions are available:
-<../upgrade>` your database. A **Rolling Release button** on your database's main screen proposes an
-upgrade to the latest version (e.g., 13.0 to 15.1).
+If the database is *not* on the latest **Online version**, the administrator should receive an
+invitation to :doc:`upgrade <../upgrade>` the database. A **Rolling Release button** on the
```
```suggestion
invitation to :doc:`upgrade <../upgrade>` the database. A :guilabel:`Rolling Release` button on the
```

StraubCreative commented on content/administration/maintain/odoo_online.rst:
```diff
@@ -35,22 +37,22 @@ Several actions are available:
-   Versions that are not supported anymore become deprecated and need to be updated to avoid
-   security issues. We recommend you initiate the upgrade of the database yourself, as this method
-   allows you to request a test upgrade of your database to check for any discrepancies.
+   Versions that are not supported anymore become deprecated and need to be updated to avoid security
+   issues. Odoo recommends that the company initiate the upgrade, as this method allows for the
+   company to request a test upgrade of the database to check for any discrepancies.
```
```suggestion
   Versions that are not supported anymore become deprecated and need to be updated to avoid
   security issues. Odoo recommends that the company initiate the upgrade, as this method allows for
   the company to request a test upgrade of the database to check for any discrepancies.
```

StraubCreative commented on content/administration/maintain/odoo_online.rst:
```diff
@@ -61,12 +63,12 @@ upgrade to the latest version (e.g., 13.0 to 15.1).
 
-Make an exact copy of the database to be able to perform testing without compromising the daily
-operations.
+This allows to make an exact copy of the database to be able to perform testing without
+compromising the daily operations.
```
```suggestion
This allows to make an exact copy of the database to be able to perform testing without compromising
the daily operations.
```

StraubCreative commented on content/administration/maintain/odoo_online.rst:
```diff
@@ -115,8 +117,8 @@ Delete a database instantly.
 
-Read carefully the warning message that pops up and proceed only if you fully understand the
-implications of deleting a database:
+Carefully read the warning message that pops up and proceed only if the implications of deleting a
+database are fully understood:
```
Reminder to break the dependency between written instruction and images; the text needs to stand on it's own without relying on images to convey important information.

Can take care of this on another PR if you want since it's outside the scope of this PR.

StraubCreative commented on content/administration/maintain/odoo_online.rst:
```diff
@@ -125,9 +127,8 @@ implications of deleting a database:
-   - It is not possible to delete a database if it is expired or linked to a subscription. If
-     needed, please get in touch with `Odoo Support <https://www.odoo.com/help>`_.
-   - To delete your account, please get in touch with `Odoo Support <https://www.odoo.com/help>`_.
+   - It is not possible to delete a database if it is expired or linked to a subscription. If needed,
+     please get in touch with `Odoo Support <https://www.odoo.com/help>`_.
```
```suggestion
   - It is not possible to delete a database if it is expired or linked to a subscription. If
     needed, please get in touch with `Odoo Support <https://www.odoo.com/help>`_.
```

StraubCreative commented on content/applications/general/users.rst:
```diff
@@ -18,3 +18,4 @@ can be applied. Users and access rights can be added and changed at any point.
    users/access_rights
    users/companies
    users/portal
+   users/delete_account
```
Tag cleanup lines 1-5 on this doc:
```
:nosearch:

===================
Users and companies
===================
```

StraubCreative reviewed (approved):
Hi @tiku-odoo 
Few small items below, which I'll fix up in the following commit and then pass over to DR.
Thanks!

StraubCreative commented:
Hi @odoo/doc-review this PR on odoo.com account deletion is ready for your review, thanks! 🙏

AntoineVDV reviewed (approved):
@robodoo r+
